webpage/index.md

```diff
@@ -6,12 +6,13 @@ layout: single
 ![Front image]({{ site.url }}/{{ site.baseurl }}/assets/Data_Demo_Center_middlepic_size770.png)
 
 The IOOS Notebook Gallery is a collection of tutorials and examples of how to access and utilize the many IOOS technologies and data sources available.
-This site is geared towards scientists and environmental managers interested in "diving deep" into the numbers and creating orginal plots and data analysis.
+This site is geared towards scientists and environmental managers interested in "diving deep" into the numbers and creating original plots and data analysis.
 Most notebooks will be examples using Python code.
-Over time, we plan to include notebooks with Matlab, R, and Arc GIS code as well.
```

Comment:
Perhaps it should just  be re-worded:  
" Most notebook examples are written in Python, however, we also have a growing number of notebooks written in Matlab, and R. "